### PR TITLE
Lps 145373 1 14

### PR DESCRIPTION
**Pull request by slnn**
Hi @dantewang @tinatian

This pull aims to make the integration test export data for SampleSQLBuilder use.

Other changes on SampleSQLBuilder are on the way.

Thanks
Lily

### CODE REVIEW

File: modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java [hunk 1]
```diff
@@ -78,7 +78,7 @@
 /**
  * @author Hai Yu
  */
-@DataGuard(scope = DataGuard.Scope.CLASS)
+@DataGuard(scope = DataGuard.Scope.NONE)
```
Comment (dantewang) [inline modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java:89]: @slnn We want this scope to be "CLASS" on CI. Please don't change it. When running it locally we can manually change it or automate the changing using other ways.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@slnn @yuhai Please see my comments and modify the pull. Thanks!

**Comment by tinatian** [comment]
@slnn I am going to close the pull based on @dantewang 's comment https://github.com/liferay-core-infra/liferay-portal/pull/200#issuecomment-1014262777

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**